react-front-end/src/components/Pagination.js

```diff
@@ -0,0 +1,41 @@
+import { useDispatch, useSelector } from 'react-redux';
+import {search} from '../actions/index'
+import Button from './Button';
+
+
+function Pagination() {
+  const dispatch = useDispatch();
+  const currentQuery = useSelector(state => state.currentQuery)
+  const pagesToRender = [];
+  const numPages = currentQuery.count/9
+  for (let i = 1; i < numPages+1; i++) {
```

Comment:
Could you do this without using a `for` loop and creating side-effects? In other words, use `map` or `reduce`?